modules/mcagar/src/main/ts/ephox/mcagar/alien/EditorTypes.ts

```diff
@@ -47,6 +47,7 @@ export interface Editor {
   setContent: (content: string) => void;
 
   execCommand: (command: string, ui?: boolean, value?: any, args?: any) => boolean;
+  setEditableRoot: (state: boolean) => void;
```

Comment:
I believe this should be optional as we don't know for sure a v6 editor actually has it
```suggestion
  setEditableRoot?: (state: boolean) => void;
```

Reply:
I can make this optional I guess and document why this is optional and add some form of check. This is a back port but since master will be merged into develop on release I can make the change here in the backport.